db/migrate/20250225131523_add_style_type_to_categories.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+class AddStyleTypeToCategories < ActiveRecord::Migration[7.2]
+  def change
+    add_column :categories, :style_type, :integer, default: 0, null: false
+    add_column :categories, :style_emoji, :string, default: ""
+    add_column :categories, :style_icon, :string, default: ""
```

Comment:
IMO we should leave these bottom 2 null, otherwise we are storing empty string for an empty value which doesn't seem right